docs/2.guide/4.recipes/4.sessions-and-authentication.md

```diff
@@ -0,0 +1,403 @@
+---
+title: 'Sessions and Authentication'
+description: "User registration and authentication is an extremely common requirement in web apps. This recipe will show you how to implement basic user registration and authentication in you Nuxt app."
+---
+
+
+## Introduction
+
+In this recipe we'll be setting up user registration, login, sessions, and authentication in a full-stack Nuxt app.
+ We'll be using [Nuxt Auth Utils](https://github.com/Atinux/nuxt-auth-utils) by [Altinux (Sébastien Chopin)](https://github.com/Atinux) which provides convenient utilities for managing front-end and back-end session data. We'll install and use this to get the core session management functionality we're going to need to manage user logins. For the database ORM we'll be using [Drizzle](https://orm.drizzle.team/) with [db0](https://db0.unjs.io/), but you can use any ORM or database connection strategy you prefer.
+
+You'll need a `users` table in your database with the following columns:
+- `id` (int, primary key, auto increment)
+- `email` (varchar)
+- `password` (varchar)
+
+Additionally, we'll use [nuxt-aut-utils](https://github.com/Atinux/nuxt-auth-utils) by [Atinux](https://github.com/Atinux) to handle the authentication and session management.
```

Comment:
Considering this was already specified in the introduction, it may not need to be repeated again.
Also it makes for inconsistent capitalization and syntax with the one in introduction as well as a typo (aut in stead of auth)

Reply:
Good feedback. Thanks.